docs/servers/index.md

```diff
@@ -10,3 +10,6 @@ links:
     related:
     featured:
 ---
+
+- [Create a GCP server](./create-gcp-server.md)
```

Comment:
Please add links to the appropriate group of links under the `links` section above.